plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountsPanel.java

```diff
@@ -0,0 +1,163 @@
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Link;
+import org.eclipse.swt.widgets.Shell;
+
+import java.util.Set;
+
+/**
+ * A panel listing all currently logged-in accounts. The panel allows adding new accounts,
+ * switching active accounts, and logging out all accounts.
+ *
+ * TODO(chanseok): move the panel to the bottom-right corner: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/740
+ */
+public class AccountsPanel extends PopupDialog {
+
+  private static final int DEFAULT_MARGIN = 5;
+  private static final int EMAIL_LEFT_MARGIN = 8;
+
+  private IGoogleLoginService loginService;
+
+  public AccountsPanel(Shell parent, IGoogleLoginService loginService) {
+    super(parent, SWT.MODELESS,
+        true /* takeFocusOnOpen */,
+        false /* persistSize */,
+        false /* persistLocation */,
+        false /* showDialogMenu */,
+        true /* showPersistActions */,
+        null /* no title area */, null /* no info text area */);
+    this.loginService = loginService;
+  }
+
+  @Override
+  protected Color getBackground() {
+    return getShell().getDisplay().getSystemColor(SWT.COLOR_WIDGET_BACKGROUND);
+  }
+
+  @Override
+  protected Color getForeground() {
+    return getShell().getDisplay().getSystemColor(SWT.COLOR_WIDGET_FOREGROUND);
+  }
+
+  @Override
+  protected Control createDialogArea(Composite parent) {
+    Composite container = (Composite) super.createDialogArea(parent);
+    GridLayoutFactory.swtDefaults().margins(DEFAULT_MARGIN, DEFAULT_MARGIN).applyTo(container);
+
+    createAccountsPane(container);
+    createButtons(container);
+    return container;
+  }
+
+  private void createAccountsPane(Composite container) {
+    Label messageLabel = new Label(container, SWT.NONE);
+    messageLabel.setText(Messages.MESSAGE_LABEL_ACTIVE_ACCOUNT);
+    FontUtil.convertFontToBold(messageLabel);
+
+    Composite activeAccountIndenter = new Composite(container, SWT.NONE);
+    GridLayoutFactory.swtDefaults().margins(EMAIL_LEFT_MARGIN, 0).applyTo(activeAccountIndenter);
+
+    Account activeAccount = loginService.getActiveAccount();
+    Label accountLabel = new Label(activeAccountIndenter, SWT.NONE);
+    accountLabel.setText(activeAccount.getEmail());
+    FontUtil.convertFontToBold(accountLabel);
+
+    Set<Account> accounts = loginService.listAccounts();
+    if (accounts.size() > 1) {
+      new Label(container, SWT.NONE).setText(Messages.MESSAGE_LABEL_OTHER_ACCOUNTS);
+
+      Composite accountsIndenter = new Composite(container, SWT.NONE);
+      GridLayoutFactory.swtDefaults().margins(EMAIL_LEFT_MARGIN, 0).applyTo(accountsIndenter);
+
+      for (Account account : accounts) {
+        if (!account.getEmail().equals(activeAccount.getEmail())) {
+          Link link = new Link(accountsIndenter, SWT.NO_FOCUS);
+          link.setText("<a href=\"" + account.getEmail() + "\">" + account.getEmail() + "</a>"); //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
Yhea, the `Link` widget uses the `<A>` element to mark a link: http://www.java2s.com/Code/Java/SWT-JFace-Eclipse/Createalinkwidget.htm